app/models/sections/info/movementScenario/MovementScenario.scala

```diff
@@ -54,31 +64,22 @@ object MovementScenario extends Enumerable.Implicits with Logging {
           MovementScenario.ExportWithCustomsDeclarationLodgedInTheEu
         }
       case DestinationType.UnknownDestination => MovementScenario.UnknownDestination
-      case answer@(DestinationType.ReturnToThePlaceOfDispatchOfTheConsignor | DestinationType.CertifiedConsignee | DestinationType.TemporaryCertifiedConsignee) =>
-        // TODO: These are Duty Paid answers which we don't currently have a design for (as of 29/11/23).
-        // We need a solid design for this, whether or not that's handled here is undecided yet, but for now we can throw an error in this case.
+      case DestinationType.ReturnToThePlaceOfDispatchOfTheConsignor => MovementScenario.ReturnToThePlaceOfDispatch
+      case DestinationType.CertifiedConsignee => MovementScenario.CertifiedConsignee
+      case DestinationType.TemporaryCertifiedConsignee => MovementScenario.TemporaryCertifiedConsignee
+      case answer =>
```

Comment:
Not sure if this is directly related to your change, but the coverage for MovementScenarioSpec doesn't show  this case as covered

Reply:
It's not possible to hit that anymore thinking about it - because the case match should be exhaustive now. So this case can be removed.